e2e-tests/spring-rest-mongo/src/test/java/org/evomaster/e2etests/spring/rest/mongo/foo/MongoEMFitnessTest.java

```diff
@@ -0,0 +1,46 @@
+package org.evomaster.e2etests.spring.rest.mongo.foo;
+
+import com.foo.spring.rest.mongo.MongoPersonsAppController;
+import org.evomaster.core.problem.rest.HttpVerb;
+import org.evomaster.core.problem.rest.RestIndividual;
+import org.evomaster.core.search.Solution;
+import org.evomaster.e2etests.utils.RestTestBase;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class
+
+
+MongoEMFitnessTest extends RestTestBase {
+
+    @BeforeAll
+    public static void initClass() throws Exception {
+        RestTestBase.initClass(new MongoPersonsAppController());
+    }
```

Comment:
class loading can be very nasty... might need to do something like `ExternalServiceMockingFlakyEMTest`, eg:

```
 @BeforeAll
    public static void initClass() throws Exception {
        ServiceController serviceController = new ServiceController();
        EMConfig config = new EMConfig();
        config.setInstrumentMR_NET(true);
        SpringTestBase.initClass(serviceController, config);
    }
```

but for `instrumentMR_MONGO`. this depends on when the SUT classes are loaded

Reply:
Both MongoFitness tests are working fine now so I don't understand if it is really necessary to change them.